effect-ts/package.json

```diff
@@ -13,7 +13,7 @@
   "peerDependencies": {
     "@hookform/resolvers": "^2.0.0",
     "@effect/schema": "^0.66.14",
-    "effect": "^3.1.2",
+    "effect": "^3.10.1",
```

Comment:
can't we remove the @effect/schema peer dependency, now?

Reply:
sure we can